src/annotator/config/settings.js

```diff
@@ -102,9 +102,22 @@ function configFuncSettingsFrom(window_) {
   return window_.hypothesisConfig();
 }
 
+/**
+ * Return true if the client is embedded in a website.
+ *
+ * @returns {boolean} true if this instance of the Hypothesis client is one
+ *   embedded in a website, false if it's one distributed in a browser
+ *   extension.
+ *
+ */
+function isEmbedded(config) {
```

Comment:
This is potentially confusing because "embedded" here also includes contexts when the client has been added by Via or the bookmarklet, but isn't really "embedded" in the page. What this test actually tells you is whether it is _served_ from the browser extension or the web.

For that reason I think it is clearer to have a test function which returns true if the client is a browser extension or false otherwise.